vpp/vaapipostprocess_base.cpp

```diff
@@ -70,6 +65,28 @@ void VaapiPostProcessBase::cleanupVA()
     m_display.reset();
 }
 
+YamiStatus VaapiPostProcessBase::queryVideoProcFilterCaps(
+    VAProcFilterType filterType, void* filterCaps, uint32_t* numFilterCaps)
+{
+    if (!filterCaps)
```

Comment:
As my view, when call this function, the caller used filterCaps == NULL && numFilterCaps == NULL is wrong, andI think you can't handle this case in the current code :)

But filterCaps != NULL && numFilterCaps == NULL or filterCaps == NULL && numFilterCaps != NULL is Ok for this function.